src/actions/setSelectionsAndFocusEditor.ts

```diff
@@ -25,3 +25,9 @@ export async function setSelectionsAndFocusEditor(
   editor.selections = selections;
   editor.revealRange(editor.selections[0]);
 }
+
+export async function focusEditor(editor: TextEditor) {
```

Comment:
Would be good to use this function on lines 21-23 above to avoid the duplication, eg `await focusEditor(editor)`

Reply:
Yes it would but I wanted to set all selections before I actually did any scrolling

Reply:
I'm confused.  This function is identical to lines 21-23 above.  Why not just remove those lines and use this function now that we have it?

Reply:
Then we are two

Reply:
Now I get what you mean. Fixed now.